lib/modules/orangered.js

```diff
@@ -171,9 +166,32 @@ const setupFaviconBadge = _.once(() => {
 	return favicon;
 });
 
-function setupFloatingButtons() {
-	if (!module.options.showFloatingEnvelope.value) return;
-	if ($floatingInboxCount) return;
+const orangeredElements = () => ({
```

Comment:
I'll sidestep that: https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3299/commits/42d273a278c85e573537a82646ec1e5cad5b5f30